docs/rfc/rfc-1.adoc

```diff
@@ -272,7 +272,7 @@ cannot pick a starting fee higher than `base_btc_fee_max` and they can never
 choose a fee higher than `btc_fee_max`.
 
 When the transaction clears, and the information has made its way
-over the relay maintainer, then another transaction needs to be created to on
+over the relay maintainer, then another transqaction needs to be created to on
```

Comment:
This got introduced accidentally, could you fix that as well? :)

Reply:
Ooops, see https://github.com/keep-network/tbtc-v2/pull/132/commits/552171477ba5f1ba08700c9246497cafb2a1882b - I squashed two last commits with typo fixes.